server/app/api/blueprints/foo.py

```diff
@@ -0,0 +1,111 @@
+from flask import Blueprint, jsonify, request
+from sqlalchemy import exc
+from app import db
+from app.api.models.foo import Foo
+
+foo_blueprint = Blueprint('foo', __name__)
+
+
+@foo_blueprint.route('/api/foo', methods=['POST'], strict_slashes=False)
+def post():
+
```

Comment:
Tiny thing, but if you're making other changes anyway could you please add a comment to these indicating they're the default response, or delete the blank line from each method?